.gitignore

```diff
@@ -1,7 +1,10 @@
 # macOS Specifics
 .DS_Store
 
-# Xcode Specifics
+# IDE Specifics
+## CLion
+.idea/
+## XCode
```

Comment:
This is fine. It's a shame there isn't better methods for organising this stuff.